extensions/resource-deployment/src/services/apiService.ts

```diff
@@ -25,6 +28,16 @@ class ApiService implements IApiService {
 		}
 		return this.azurecoreApi;
 	}
+
+	public async getArcApi(): Promise<arc.IExtension> {
+		if (!this.arcApi) {
+			this.arcApi = <arc.IExtension>(await vscode.extensions.getExtension(arc.extension.name)?.activate());
```

Comment:
Don't do this - just use the exports instead (When I implemented the azurecore version of this I didn't know enough about the area - but doing activate here is pretty much unnecessary since if it isn't activated there's no way we'd be able to do anything anyways so requiring the async call is just annoying to deal with)